src/main/resources/resources/bric/ColorPicker_he.properties

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2008 Jeremy Wood
+# E-mail: mickleness@gmail.com
+# All rights reserved.
+#
+# The copyright of this software is owned by Jeremy Wood.
+# You may not use, copy or modify this software, except in
+# accordance with the license agreement you entered into with
+# Jeremy Wood. For details see accompanying license terms.
+#
```

Comment:
Not sure this header is needed, as you (@YovelB) did the localisation and just used the labels.

Reply:
Removing the file altogether seems like the right choice for now.